app-crypt/tpm2-tss/files/tpm2-tss-3.2.0-test-fix-usage-of-FILE-in-unit-test-fapi-io.patch

```diff
@@ -0,0 +1,142 @@
+From bda22252507124bb8e466ac2f0c61d5ebed9027d Mon Sep 17 00:00:00 2001
```

Comment:
Link to upstream commit?

Reply:
Link here: https://github.com/tpm2-software/tpm2-tss/commit/bda22252507124bb8e466ac2f0c61d5ebed9027d PR: https://github.com/tpm2-software/tpm2-tss/pull/2423

Reply:
I'll add it to the top of the patch, thanks!